packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -97,8 +79,8 @@ export class GCTelemetryTracker {
 	// Keeps track of unreferenced events that are logged for a node. This is used to limit the log generation to one
 	// per event per node.
```

Comment:
per session? Because it's not cleared in `gcRunCompleted`. Would be nice to confirm/clarify